Core/Utilities/Subprocess.py

```diff
@@ -34,7 +34,10 @@
 import select
 import os
 import sys
-import subprocess
+try:
+  import subprocess32 as subprocess
```

Comment:
This is a backport of the Python 3 subprocess module which fixes many subtle bugs. It is already included in diracos as it's a dependency of lots of other packages.

In this case I changed it as running `Popen(['ps', ...])` is 100x faster when the load on the machine is high (from 13.5 seconds to 150ms). To give an idea of how big a difference it made, this reduced the time taken to run the integration tests in CI from ~1.5 hours to 20 minutes.

Shall I also replace the other uses of `subprocess`?